webapp/javascript/pages/tagExplorer/components/Legend.tsx

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
 
 import type { TimelineGroupData } from '@webapp/components/TimelineChartWrapper';
+import { appWithoutTagsWhereDropdownOptionName } from '@webapp/redux/reducers/continuous';
 import styles from './Legend.module.scss';
 
 interface LegendProps {
   groups: TimelineGroupData[];
   handleGroupByTagValueChange: (groupByTagValue: string) => void;
+  activeGroup: string;
 }
 
-function Legend({ groups, handleGroupByTagValueChange }: LegendProps) {
+function Legend({
+  groups,
+  handleGroupByTagValueChange,
+  activeGroup,
+}: LegendProps) {
   return (
     <div data-testid="legend" className={styles.legend}>
       {groups.map(({ tagName, color }) => (
         <div
+          aria-hidden
           data-testid="legend-item"
           className={styles.tagName}
           key={tagName}
-          onClick={() => handleGroupByTagValueChange(tagName)}
+          onClick={() =>
+            handleGroupByTagValueChange(
```

Comment:
I'm not sure whats going on here can you explain / make this code a littler clearer? 

for example could this be simplified to a variable `isSelected`